### ISSUE TITLE
Remove unused branches?

### ISSUE THREAD

**Issue by ssardina** [SWI-Prolog/packages-jpl#75]
Hi @JanWielemaker 

Can we totally remove the `stable` and `cmake` branches? THey are old and I think they are either behind or have changes already accounted for (like doc)

**Comment by JanWielemaker** [comment]
Yes.  The stable branch will be recreated at the moment the stable and development version use a different JPL version.   Just deleted.   There are several others, where I do not know whether they are half baked improvements, failed attempts or merged topic branches :(

**Comment by ssardina** [comment]
Great, thanks for cleaning the up. 

I have removed before all those that were merged. There is only one left `extend-engine-pool` that is meant to change the C code to handle the pool of Prolog engines more flexible, it is a big enhancement, nothing core, and it is half baked. 

But it is related to an issue #25 , so I renamed it to make it clear the link. 

Unless it is short lived, one should try to link branches with specific issues, precisely to avoid losing track of what is happening! :-)
